DataDefinitions/StationModels.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Utilities;
+
+namespace EddiDataDefinitions
+{
+    /// <summary>
+    /// Crime types
```

Comment:
Aaaaand this is why I don't like docstrings. They get stale and tell lies. I might just nuke the lot of them, they are completely redundant anyway.

Reply:
woops. well, I guess you know where I found my template. :-)